core/src/com/unciv/logic/map/MapUnit.kt

```diff
@@ -450,8 +450,10 @@ class MapUnit {
 
     private fun clearEncampment(tile: TileInfo) {
         tile.improvement=null
-        val goldToAdd = 25 // game-speed-dependant
-        civInfo.gold+=goldToAdd
+        var goldToAdd = 25 // game-speed-dependant
```

Comment:
Comment says that it should multiply on game-speed factor, but I don't see it here? 
I'm still don't know codebase well, just asking)